lib/ueberauth/strategy/facebook.ex

```diff
@@ -181,11 +175,11 @@ defmodule Ueberauth.Strategy.Facebook do
   end
 
   defp option(conn, key) do
-    default = Keyword.get(default_options(), key)
+    default = Dict.get(default_options, key)
```

Comment:
Yeah, true. I was doing this on hackathon as far as I remember thus I put there whatever google spilled first.